openedx/core/djangoapps/coursetalk/tests/test_helpers.py

```diff
@@ -0,0 +1,107 @@
+""" CourseTalk widget helpers tests """
+from __future__ import unicode_literals
+
+from unittest import skipUnless
+
+from django import test
+from django.conf import settings
+
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from openedx.core.djangoapps.coursetalk import helpers
+from openedx.core.djangoapps.coursetalk import models
+
+
+class GetCoursetalkCourseKeyTests(test.TestCase):
+    """
+    GetCoursetalkCourseKeyTests:
+    tests for function get_coursetalk_course_key
+    """
+
+    def test_simple_key(self):
+        course_key = SlashSeparatedCourseKey('org', 'course', 'run')
+        coursetalk_course_key = helpers.get_coursetalk_course_key(course_key)
+        self.assertEqual(coursetalk_course_key, 'org_course')
+
+
+@skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Tests only valid in LMS')
+class InjectCoursetalkKeysIntoContextBaseTests(test.TransactionTestCase):
+    """
+    InjectCoursetalkKeysIntoContext:
+    tests for function inject_coursetalk_keys_into_context
+    """
+
+    ENABLED = False
+    PLATFORM_KEY = 'some_platform'
+
+    @classmethod
+    def setUpClass(cls):
+        super(InjectCoursetalkKeysIntoContextBaseTests, cls).setUpClass()
+        cls.course_key = SlashSeparatedCourseKey('org', 'course', 'run')
+        cls.context = {}
+        cls.db_set_up()
+        cls.inject()
+
+    @classmethod
+    def db_set_up(cls):
+        """
+        Setup database for this test:
+        Create CourseTalkWidgetConfiguration
+        """
+        config = models.CourseTalkWidgetConfiguration.current()
+        config.enabled = cls.ENABLED
+        config.platform_key = cls.PLATFORM_KEY
+        config.save()
+
+    @classmethod
+    def inject(cls):
+        """
+        Run tested function: inject_coursetalk_keys_into_context
+        """
+        helpers.inject_coursetalk_keys_into_context(cls.context, cls.course_key)
+
+
+@skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Tests only valid in LMS')
+class InjectCoursetalkKeysIntoContextWhenWidgetNotEnabled(InjectCoursetalkKeysIntoContextBaseTests):
+    """
+    InjectCoursetalkKeysIntoContextWhenWidgetNotEnabled:
+    tests for function inject_coursetalk_keys_into_context
+    when widget not enabled
+    """
+
+    def test_show_coursetalk_widget_not_in_context(self):
+        self.assertNotIn('show_coursetalk_widget', self.context)
+
+    def test_platform_key_not_in_context(self):
+        self.assertNotIn('platform_key', self.context)
+
+    def test_course_review_key_not_in_context(self):
+        self.assertNotIn('course_review_key', self.context)
+
+
+@skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Tests only valid in LMS')
+class InjectCoursetalkKeysIntoContextWhenWidgetEnabled(InjectCoursetalkKeysIntoContextBaseTests):
```

Comment:
Both of these inheriting classes should probably just be rolled up into the `GetCourseTalkKeyTests` class -- I don't see any reason for them to be separated. You could name it something generic like `CourseTalkKeyTests`.